bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TT_SStt/TT_SStt_proc_card.dat

```diff
@@ -0,0 +1,185 @@
+set mass 6100001 50
+done
+launch
+set mass 6000006 800
+set mass 6100001 100
+launch
+set mass 6000006 800
+set mass 6100001 150
+launch
+set mass 6000006 800
+set mass 6100001 200
+launch
+set mass 6000006 800
+set mass 6100001 250
+launch
+set mass 6000006 800
+set mass 6100001 300
+launch
+set mass 6000006 800
+set mass 6100001 350
+launch
+set mass 6000006 900
+set mass 6100001 50
+launch
+set mass 6000006 900
+set mass 6100001 100
+launch
+set mass 6000006 900
+set mass 6100001 150
+launch
+set mass 6000006 900
+set mass 6100001 200
+launch
+set mass 6000006 900
+set mass 6100001 250
+launch
+set mass 6000006 900
+set mass 6100001 300
+launch
+set mass 6000006 900
+set mass 6100001 350
+launch
+set mass 6000006 1000
+set mass 6100001 50
+launch
+set mass 6000006 1000
+set mass 6100001 100
+launch
+set mass 6000006 1000
+set mass 6100001 150
+launch
+set mass 6000006 1000
+set mass 6100001 200
+launch
+set mass 6000006 1000
+set mass 6100001 250
+launch
+set mass 6000006 1000
+set mass 6100001 300
+launch
+set mass 6000006 1000
+set mass 6100001 350
+launch
+set mass 6000006 1100
+set mass 6100001 50
+launch
+set mass 6000006 1100
+set mass 6100001 100
+launch
+set mass 6000006 1100
+set mass 6100001 150
+launch
+set mass 6000006 1100
+set mass 6100001 200
+launch
+set mass 6000006 1100
+set mass 6100001 250
+launch
+set mass 6000006 1100
+set mass 6100001 300
+launch
+set mass 6000006 1100
+set mass 6100001 350
+launch
+set mass 6000006 1100
+set mass 6100001 350
+launch
+set mass 6000006 1200
+set mass 6100001 50
+launch
+set mass 6000006 1200
+set mass 6100001 100
+launch
+set mass 6000006 1200
+set mass 6100001 150
+launch
+set mass 6000006 1200
+set mass 6100001 200
+launch
+set mass 6000006 1200
+set mass 6100001 250
+launch
+set mass 6000006 1200
+set mass 6100001 300
+launch
+set mass 6000006 1200
+set mass 6100001 350
+launch
+set mass 6000006 1300
+set mass 6100001 50
+launch
+set mass 6000006 1300
+set mass 6100001 100
+launch
+set mass 6000006 1300
+set mass 6100001 150
+launch
+set mass 6000006 1300
+set mass 6100001 200
+launch
+set mass 6000006 1300
+set mass 6100001 250
+launch
+set mass 6000006 1300
+set mass 6100001 300
+launch
+set mass 6000006 1300
+set mass 6100001 350
+launch
+set mass 6000006 1400
+set mass 6100001 50
+launch
+set mass 6000006 1400
+set mass 6100001 100
+launch
+set mass 6000006 1400
+set mass 6100001 150
+launch
+set mass 6000006 1400
+set mass 6100001 200
+launch
+set mass 6000006 1400
+set mass 6100001 250
+launch
+set mass 6000006 1400
+set mass 6100001 300
+launch
+set mass 6000006 1400
+set mass 6100001 350
+launch
+set mass 6000006 1500
+set mass 6100001 50
+launch
+set mass 6000006 1500
+set mass 6100001 100
+launch
+set mass 6000006 1500
+set mass 6100001 150
+launch
+set mass 6000006 1500
+set mass 6100001 200
+launch
+set mass 6000006 1500
+set mass 6100001 250
+launch
+set mass 6000006 1500
+set mass 6100001 300
+launch
+set mass 6000006 1500
+set mass 6100001 350
+launch
```

Comment:
for all the mass point, you can make a separate script to implement them instead of putting them together with proc card